app/Services/Model/Imp/MarketingService.php

```diff
@@ -0,0 +1,76 @@
+<?php namespace App\Services\Model\Imp;
+/**
+ * Copyright 2024 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+use App\Services\Model\AbstractService;
+use App\Services\Model\IMarketingService;
+use Illuminate\Support\Facades\Log;
+use libs\utils\ICacheService;
+use libs\utils\ITransactionService;
+use services\apis\IMarketingAPI;
+
+/**
+ * Class MarketingService
+ * @package App\Services\Model\Imp
+ */
+final class MarketingService
+    extends AbstractService
+    implements IMarketingService
+{
+    const CacheTTL = 60;
```

Comment:
dont create a new service
just use the api and do the cache on the api facade
check https://github.com/OpenStackweb/summit-api/blob/9f48390f7f3d33ab15963f3558722ce4049c9527/app/Services/Apis/MailApi.php#L56

Reply:
done